webhook/resourcesemantics/validation/validation_admit.go

```diff
@@ -179,14 +195,14 @@ func (ac *reconciler) callback(ctx context.Context, req *admissionv1beta1.Admiss
 
 	// Generically callback if any are provided for the resource.
 	if c, ok := ac.callbacks[gvk]; ok {
-		if b, supported := c.SupportedVerbs[req.Operation]; supported && b {
+		if b, supported := c.supported[req.Operation]; supported && b {
```

Comment:
you only put `true` in the map, so just presence, i..e `supported == true` is enough.

Reply:
sure. perhaps easier to read